### PR TITLE
Fix some DM Deletion styling issues

### PR DESCRIPTION
**Pull request by clcaelynn-clever**
# Overview:

Fix some DM Deletion styling issues, pertaining to the Delete/timestamp action button in NormalMessagingBubble.

# Testing:

Tested in both local components instance as well as in local launchpad.

- [ ] Unit tests
- Manual tests:
  - [x] Chrome
  - [x] Safari
  - [ ] IE11

# Roll Out:

- Before merging:
  - [n/a] Updated docs
  - [x] Bumped version in `package.json`
    - Breaking change?
      - If it is a beta component run `npm version minor`
      - If the component is not in beta run `npm version major`
    - New component or backward-compatible component feature change? Run `npm version minor`
    - Only changing documentation? All good. Skip this step.
  - After creating a new component, make sure to add it to the Components List in `ComponentsView.jsx`. To do so:
    - [ ] Add a screenshot of the component in `docs/assets/img` with the format `<COMPONENT URL LINK>.png`
- After merging:
  - [ ] Deployed updated docs (`make deploy-docs`)
  - [ ] Posted in #eng if I made a breaking change to a beta component

### CODE REVIEW

File: src/MessagingBubble/NormalMessagingBubble.less [hunk 1]
```diff
@@ -79,11 +79,15 @@
 }
 
 .NormalMessagingBubble--Message--Metadata--Own {
+  min-width: 3.5rem;
```
Comment (clcaelynn-clever) [inline src/MessagingBubble/NormalMessagingBubble.less:82]: Avoids squashing in small'ish viewport width.

File: src/MessagingBubble/NormalMessagingBubble.less [hunk 2]
```diff
@@ -140,7 +144,7 @@
   }
 }
 
-Button.NormalMessagingBubble--Message--ActionButton--Own {
+.Button.NormalMessagingBubble--Message--ActionButton--Own {
```
Comment (clcaelynn-clever) [inline src/MessagingBubble/NormalMessagingBubble.less:147]: 🙃

File: src/MessagingBubble/NormalMessagingBubble.less [hunk 3]
```diff
@@ -79,11 +79,15 @@
 }
 
 .NormalMessagingBubble--Message--Metadata--Own {
+  min-width: 3.5rem;
   .margin--right--xs();
+  text-align: right;
```
Comment (clcaelynn-clever) [inline src/MessagingBubble/NormalMessagingBubble.less:84]: Ensures closest-to-message alignment for timestamp and "Delete" text.

**Review by jonjonray — APPROVED** [review]
LGTM!

**Pull request merged**